### PR TITLE
Moved the Windows Sensu client tasks

### PR DESCRIPTION
**Pull request by lbussey**
Moved Windows-based tasks to a directory following the same conventions as the Linux platforms. This change coincides with the updates to the sensu-client/master-deployment changes to add Linux support

https://github.com/copperleaftech/sensu-master-deployment/pull/73
https://github.com/copperleaftech/sensu-client-deployment/pull/20

### CODE REVIEW

File: tasks/Windows/main.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  - name: Ensure Temp Directory exists
+    win_file:
+      path: "{{ sensu_win_temp_path }}"
+      state: directory
+
+  - name: Download the Sensu installer msi into a local dir
+    win_get_url:
+      url: "{{ sensu_msi_download_path }}"
+      dest: "{{ sensu_win_temp_path }}\\sensu-installer.msi"
+      force: yes
```
Comment (gjohannsen-copperleaf) [inline tasks/Windows/main.yml:16]: Do we need a force here? I believe this downloads the installer again every time we run the deployment?
I think we would only need to set this to true if the installer was changing.

Comment (gjohannsen-copperleaf) [reply tasks/Windows/main.yml:16]: Maybe do a clever when here

Comment (lbussey) [reply tasks/Windows/main.yml:16]: Is this something we should change within the scope of this current project?

Comment (gjohannsen-copperleaf) [reply tasks/Windows/main.yml:16]: Can be fixed other time

**Comment by sstaszkiewicz-copperleaf** [comment]
This PR looks stale. Can you close and reopen when it's ready so I stop getting alerted about it every day? 😄

**Comment by gjohannsen-copperleaf** [comment]
Closing this as we are moving to sensu go

**Pull request closed without merging**